frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -130,21 +148,27 @@ describe('CRD extensions', () => {
           expect(crudView.resourceTitle.getText()).toEqual(instanceName);
         });
 
-        it(`displays the ${crd} instance in the ${dropdownMenuName}`, async () => {
-          await browser.get(`${appHost}`);
-          // reload the app so the new CRD link is visible
-          await browser.refresh();
-          await browser.wait(until.presenceOf(dropdownToggle));
-          await browser.wait(dropdownToggle.click());
-          await browser.wait(
-            until.presenceOf(
-              element(by.cssContainingText('.pf-c-app-launcher__menu-item', menuLinkText)),
-            ),
-          );
-          expect(
-            element(by.cssContainingText('.pf-c-app-launcher__menu-item', menuLinkText)).getText(),
-          ).toContain(menuLinkText);
-        });
+        it(
+          `displays the ${crd} instance in the ${dropdownMenuName}`,
+          async () => {
+            await browser.get(`${appHost}`);
+            // reload the app so the new CRD link is visible
+            await browser.refresh();
+            await browser.wait(until.presenceOf(dropdownToggle));
+            await browser.wait(dropdownToggle.click());
+            await browser.wait(
+              until.presenceOf(
+                element(by.cssContainingText('.pf-c-app-launcher__menu-item', menuLinkText)),
+              ),
+            );
+            expect(
+              element(
+                by.cssContainingText('.pf-c-app-launcher__menu-item', menuLinkText),
+              ).getText(),
+            ).toContain(menuLinkText);
+          },
+          JASMSPEC_TIMEOUT,
```

Comment:
Can we change the default timeout if needed instead of passing it to every test? I would think we only need to set it in rare cases for tests that are very slow (for instance, some of the OLM tests that wait for an operator to install).

Reply:
Thanks for reviewing promptly. That's something I can definitely look into. I tried it in `jasmineNodeOps` in `protractor.conf`, but somehow that didn't help (at that time). I'll give it another try.

Reply:
Don't know how I had given up this better approach, but it seems to be working as expected. PTAL.